config/locales/en.yml

```diff
@@ -75,7 +75,15 @@ en:
         para_3: "Questions? Reply to this email."
         sign_off: "Sincerely,"
         from: "The 18F Micro-purchase team"
-
+      winning_bidder_missing_payment_method:
+        subject: "Please update your profile so Micro-purchase can pay you for your great work!"
+        para_1: "A payment of %{amount} is pending release regarding work you
+          recently completed on \"%{auction_title}\". In order to take receipt of payment, you
```

Comment:
minor thing: `In order to take receipt of payment` -> `In order for us to pay you`

We should probably link to _something_ or add a sentence or two about what this means. Thoughts @mtorres253? Could be done in a subsequent PR, too.